jinx.el

```diff
@@ -858,6 +859,17 @@ If prefix argument ALL non-nil correct all misspellings."
         (goto-char old-point)
         (jinx--in-base-buffer #'jit-lock-refontify (window-start) (window-end))))))
 
+;;;###autoload
+(defun jinx-correct-word-at-point ()
+  "Correct word at cursor.
+Suggest corrections even if it's not misspelled."
+  (interactive)
+  (unless jinx-mode (jinx-mode 1))
+  (pcase-let ((`(,beg . ,end) (bounds-of-thing-at-point 'word)))
```

Comment:
Instead of `bounds-of-thing-at-point` we should use `\<\w+\>` and the `jinx--syntax-table`. See `jinx--check-region`.

Reply:
> Instead of `bounds-of-thing-at-point` we should use `\<\w+\>` and the `jinx--syntax-table`. See `jinx--check-region`.

Done at https://github.com/minad/jinx/pull/89/commits/f2a901f3f3549ddff45854b260e2ab78ed752f0a, but I am not sure if it's correct.

Reply:
Thanks. At first it is likely not correct since you don't unset the syntax table for example. It also looks a bit too complex. But do not bother with that right now. I think it is more important to get the overall structure right at first.